katas/content/index.json

```diff
@@ -15,5 +15,6 @@
   "oracles",
   "deutsch_algo",
   "deutsch_jozsa",
-  "qec_shor"
+  "qec_shor",
+  "marking_oracles"
```

Comment:
We don't need to do this any longer! I did a change last week that builds non-published katas too and adds them to the playground, so no more adding-then-deleting it, just the final add when ready to publish :-)